docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2131,14 +2131,25 @@ sync-mode="FAST"
 The synchronization mode. The options are `FAST` and `FULL`. The default is `FULL`.
 
 !!! note
-
-    If fast synchronization fails, usually because the node can't find enough valid peers, fast
-    synchronization restarts after a short delay.
+    
+    On AWS, a known [RocksDB](https://github.com/facebook/rocksdb/issues/6435) issue causes fast sync 
```

Comment:
Should we change "On AWS" for "When running Besu on certain cloud providers, notably AWS and Digital Ocean,"

Reply:
Updated